tv_apps/device-deck/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+    <meta charset="utf-8">
+    <title>Devices</title>
+
+    <link rel="stylesheet" type="text/css" href="style/devices.css">
```

Comment:
I removed them on purpose since we don't have any strings now. Would it cause any problem in build time?